client/src/containers/routing/index.tsx

```diff
@@ -43,7 +43,7 @@ const Routing: React.FC<IRoutingProps> = () => {
 			/>
 			<PublicRoute exact restricted={true} path={ROUTES.Landing} component={LandingPageCointainer} />
 			<PrivateRoute path="/private" component={TestPrivate} needHeader={false} needSideBar={false} />
-			<PrivateRoute exact path={ROUTES.Home} component={HomePage} needHeader={true} needSideBar={true} />
+			<PublicRoute exact restricted={true} path={ROUTES.Home} component={HomePage} needHeader={true} needSideBar={true} />
```

Comment:
Why this is public route?
Check https://www.codewars.com/dashboard
You can't enter it without being logged in.

I approve this PR but please, return to PrivateRoute

Reply:
Check out route rules here too
https://trello.com/c/djSXb81Q/92-routes